lib/cosmos/interfaces/adapters/override.rb

```diff
@@ -0,0 +1,44 @@
+# encoding: ascii-8bit
+
+# Copyright 2014 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+module Cosmos
+
+  module Override
+    def override(target_name, packet_name, item_name, value)
```

Comment:
should be the same name as the api method.   nil can be returned, so probably do also need a clear_tlm_override method